internal/kms/azure_vault.go

```diff
@@ -106,19 +105,11 @@ func initAzureKeyVaultKMS(args ProviderInitArgs) (EncryptionKMS, error) {
 		return nil, fmt.Errorf("failed to get secrets for %T, %w", kms, err)
 	}
 
-	var encodedClientCertificate string
-	err = setConfigString(&encodedClientCertificate, secrets, azureClientCertificate)
+	err = setConfigString(&kms.clientCertificate, secrets, azureClientCertificate)
 	if err != nil {
 		return nil, err
 	}
 
-	clientCertificate, err := base64.StdEncoding.DecodeString(encodedClientCertificate)
-	if err != nil {
-		return nil, fmt.Errorf("failed to decode client certificate: %w", err)
-	}
-
-	kms.clientCertificate = string(clientCertificate)
```

Comment:
This assignment is not required anymore?

Reply:
No, not required.
assigning it here 
```go
err = setConfigString(&kms.clientCertificate, secrets, azureClientCertificate)
```